modules/fundamental/tests/sbom/issue_1535.rs

```diff
@@ -0,0 +1,20 @@
+use test_context::test_context;
+use test_log::test;
+use trustify_module_ingestor::service::Format;
+use trustify_test_context::TrustifyContext;
+
+/// This is a test for issue #1535
+#[test_context(TrustifyContext)]
+#[test(tokio::test)]
+async fn not_really_clearly_defined(ctx: &TrustifyContext) -> Result<(), anyhow::Error> {
+    let result = ctx
+        .ingest_document_as("csaf/timeout/rhsa-2024_5363.json.xz", Format::SBOM)
```

Comment:
I'm curious why the Format needs to be passed. What happens with...
```suggestion
        .ingest_document("csaf/timeout/rhsa-2024_5363.json.xz")
```

Reply:
fair question I'm simulating the upload using `sbom` tab, and using a `csaf` file so that the code flow will reach this parthttps://github.com/trustification/trustify/blob/main/modules/ingestor/src/service/format.rs#L135 and in case using `serde_yml` we can see the `panic` happening

Reply:
the panic here: https://github.com/trustification/trustify/issues/1535#issuecomment-2801665857